.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```

Comment:
You might need to add a directory wild card like  `**/node_modules` so it will apply at any directory level. Also, it might need a closing slash to indicate you mean the entire folder like `**/node_modules/` (but I'm not 100% on that, I could be confusing syntax)